Assets/Scripts/Managers/SpawningManager.cs

```diff
@@ -43,12 +44,20 @@ public class SpawningManager : MonoBehaviour
     private Vector3 area;
     private Collider[] hitColliders;
 
+    private TextMeshProUGUI planetName;
+
+    public static SpawningManager Instance;
```

Comment:
This instance is not getting assigned. Is it needed?

Reply:
It's been removed : )